src/Core/src/Handlers/View/ViewHandler.Standard.cs

```diff
@@ -2,26 +2,82 @@
 {
 	public partial class ViewHandler
 	{
+		/// <summary>
+		/// Maps a view's abstract <see cref="ITransform.TranslationX"/> property to the platform-specific implementations.
```

Comment:
Worth noting that `IView` inherits from `ITransform`, so I can't `cref IView.TranslationX`. 

I wasn't really sure how to word the docs any other way. Happy to take any feedback here.